web/src/HUD.tsx

```diff
@@ -7,14 +7,11 @@ import Statusbar, { StatusItem } from "./Statusbar"
 import LogPane from "./LogPane"
 import K8sViewPane from "./K8sViewPane"
 import PreviewPane from "./PreviewPane"
-import { Map } from "immutable"
-import {
-  BrowserRouter as Router,
-  Route,
-  Switch,
-  RouteComponentProps,
-} from "react-router-dom"
+import { Map as ImmMap } from "immutable"
```

Comment:
hmmm...this doesn't seem right, we want to use the immutable map by default

Reply:
the name conflicted with the ES6 map which I was very confused by. The solution might just be to use objects for analytics instead of maps in the first place.